packages/tenant-process/src/services/tenantService.ts

```diff
@@ -1711,6 +1712,42 @@ export function tenantServiceBuilder(
         )
       );
     },
+    async removeTenantDelegatedConsumerFeature({
+      organizationId,
+      correlationId,
+      logger,
+    }: {
+      organizationId: TenantId;
+      correlationId: CorrelationId;
+      logger: Logger;
+    }): Promise<void> {
+      logger.info(
+        `Removing delegated consumer feature to tenant ${organizationId}`
+      );
+
```

Comment:
IPA check shall be added here as well as done for the same method for the producer delegations feature: https://vscode.dev/github/pagopa/interop-be-monorepo/blob/develop/packages/tenant-process/src/services/tenantService.ts#L1689